src/commands/generate/index.ts

```diff
@@ -1,12 +1,17 @@
 import { Command } from '@oclif/core';
 import { prompt } from 'inquirer';
 
-import { getProjectPath } from '../../helpers/file';
+import { getProjectPath, remove } from '../../helpers/file';
 import { detectTerraform, formatCode } from '../../helpers/terraform';
+import {
+  applyVersionControl,
+  versionControlChoices,
+} from '../../templates/addons/versionControl';
 import { generateAwsTemplate } from '../../templates/aws';
 
 type GeneralOptions = {
   projectName: string;
+  versionControl?: 'github' | 'gitlab';
```

Comment:
```suggestion
  versionControl?: 'github' | 'none';
```
(or eventually "other")

![image](https://user-images.githubusercontent.com/77609814/183586343-5b1592e6-55a9-4d8e-9183-00b74ffbdf80.png)

Reply:
😅 Removed in https://github.com/nimblehq/infrastructure-templates/pull/97/commits/61b304a1139e79660f96902fa0b5f539be5eab9f